pkg/client/kubernetes/client.go

```diff
@@ -76,6 +76,28 @@ func NewRuntimeClientForConfig(fns ...ConfigFunc) (client.Client, error) {
 	return client.New(conf.restConfig, conf.clientOptions)
 }
 
+// NewRESTConfigFromSecret creates a new REST config for a given secret.
+func NewRESTConfigFromSecret(secret *corev1.Secret) (*rest.Config, error) {
```

Comment:
See below - these functions are probably not needed and can be removed again.

Reply:
Removed.